pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -109,13 +109,13 @@ class CreateVM extends React.Component {
     }
 
     onChangedEventValue(key, e) {
-        if (e && e.target && typeof(e.target.value) !== 'undefined') {
+        if (e && e.target && typeof e.target.value !== 'undefined') {
```

Comment:
IMO, this makes code harder to read.
Probably introduced by https://eslint.org/docs/rules/space-unary-ops

Reply:
We use `typeof` without the extra parentheses a lot (and it's an operator, so it's syntactically valid). I'm fine with adding them back and instead having `typeof (foo)`, that's also ok.

Reply:
ok